src/cow-react/common/hooks/useThrottleFn.ts

```diff
@@ -0,0 +1,18 @@
+import { useCallback, useState } from 'react'
+
+export function useThrottleFn(fn: (...args: unknown[]) => unknown, timeout: number): typeof fn {
```

Comment:
We have an indirect dependency with a throttle https://www.npmjs.com/package/@react-hook/throttle

<img width="545" alt="image" src="https://user-images.githubusercontent.com/2352112/204743690-e659611a-4c91-4b91-ade4-cd7a5b4bcb38.png">

Reply:
Is this hook useful here?

Reply:
Maybe it's useful, anyway, we don't have the dependency in our `package.json`. I think it's not a good idea to use indirect deps

Reply:
yep, my point was more, if its a good option, and is already in our bundle, we can consider adding as dependency.